src/ovirtapi/transform.js

```diff
@@ -661,10 +655,79 @@ const StorageDomain = {
   toInternal ({ storageDomain }: { storageDomain: ApiStorageDomainType }): StorageDomainType {
     const permissions = Permissions.toInternal({ permissions: storageDomain?.permissions?.permission })
 
+    // frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/dataprovider/AsyncDataProvider.java#getDiskVolumeFormat
```

Comment:
please create a follow up patch to Webadmin and add a comment to both mentioned methods  i.e. sth similar to 
> This logic is duplicated int <...> Please update on any change.
> TODO: share this logic with VM Portal via REST API